tools/amd_build/build_amd.py

```diff
@@ -35,7 +35,12 @@
     "c10/cuda/*",
     # PyTorch paths
     # Keep this synchronized with is_pytorch_file in hipify_python.py
-    "aten/*",
+    "aten/src/ATen/cuda/*",
```

Comment:
The downside is if someone add cuda files in a new directory in aten then they will see hip error which might feel confusing. How much of time got saved with this pruning?